src/Cursors.ts

```diff
@@ -105,7 +105,7 @@ export default class Cursors extends EventEmitter<CursorsEventMap> {
     const channel = this.getChannel();
     const cursorsMembers = await channel.presence.get();
     this.cursorBatching.setShouldSend(cursorsMembers.length > 1);
-    this.cursorBatching.setBatchTime(cursorsMembers.length * this.options.outboundBatchInterval);
+    this.cursorBatching.setBatchTime(Math.ceil(cursorsMembers.length / 100) * this.options.outboundBatchInterval);
```

Comment:
It's hard to understand why this is divided by 100 now?

Reply:
Ok thanks, makes sense, perhaps put that as a comment in the code?

Reply:
Added as a comment in the code